source/first-instance/dashboard.rst

```diff
@@ -139,8 +151,17 @@ as the boot source and "No" for create new volume. Select the
 .. image:: ../_static/fi-launch-instance-source.png
    :align: center
 
+|
+
+.. important:: When creating a windows instance you need to select the ``windows-server-2012r2-x86_64`` image from the image list first, then select "Yes" to create new volume.  An alert symbol will appear alongside the Size(GB) selector.  If you hover the mouse on this it will show the minimum size of the volume required to host the windows image.  Set an appropriate volume size, select "Yes" to Delete Volume on Terminate then click "Next".
```

Comment:
Don't think it is appropriate for us to recommend customers must select "Yes" on Delete Volume on Terminate. I think this needs to be there decision and we need to explain the consequences of doing so. Could we remove this from this note?